github.sgnl.yaml

```diff
@@ -0,0 +1,1118 @@
+        externalId: $.node.companyHTML
+        type: String
+      - name: createdAt
+        externalId: $.node.createdAt
+        type: DateTime
+      - name: databaseId
+        externalId: $.node.databaseId
+        type: Int64
+      - name: email
+        externalId: $.node.email
+        type: String
+      - name: login
+        externalId: $.node.login
+        type: String
+      - name: isBountyHunter
+        externalId: $.node.isBountyHunter
+        type: Bool
+      - name: isCampusExpert
+        externalId: $.node.isCampusExpert
+        type: Bool
+      - name: isDeveloperProgramMember
+        externalId: $.node.isDeveloperProgramMember
+        type: Bool
+      - name: isEmployee
+        externalId: $.node.isEmployee
+        type: Bool
+      - name: isFollowingViewer
+        externalId: $.node.isFollowingViewer
+        type: Bool
+      - name: isHireable
+        externalId: $.node.isHireable
+        type: Bool
+      - name: isSiteAdmin
+        externalId: $.node.isSiteAdmin
+        type: Bool
+      - name: isViewer
+        externalId: $.node.isViewer
+        type: Bool
+      - name: location
+        externalId: $.node.location
+        type: String
+      - name: name
+        externalId: $.node.name
+        type: String
+      - name: pinnedItemsRemaining
+        externalId: $.node.pinnedItemsRemaining
+        type: Int64
+      - name: projectsResourcePath
+        externalId: $.node.projectsResourcePath
+        type: String
+      - name: projectsUrl
+        externalId: $.node.projectsUrl
+        type: String
+      - name: resourcePath
+        externalId: $.node.resourcePath
+        type: String
+      - name: twitterUsername
+        externalId: $.node.twitterUsername
+        type: String
+      - name: updatedAt
+        externalId: $.node.updatedAt
+        type: DateTime
+      - name: url
+        externalId: $.node.url
+        type: String
+      - name: viewerCanChangePinnedItems
+        externalId: $.node.viewerCanChangePinnedItems
+        type: Bool
+      - name: viewerCanCreateProjects
+        externalId: $.node.viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanFollow
+        externalId: $.node.viewerCanFollow
+        type: Bool
+      - name: viewerIsFollowing
+        externalId: $.node.viewerIsFollowing
+        type: Bool
+      - name: websiteUrl
+        externalId: $.node.websiteUrl
+        type: String
+  CollaboratorOVDE:
+    parent: Collaborator
+    displayName: CollaboratorOVDE
+    externalId: $.node.organizationVerifiedDomainEmails
+    description: Entity representing a GitHub user's organization verified domain emails.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: email
+        externalId: email
+        type: String
+        indexed: true
+        uniqueId: true
+relationships:
```

Comment:
@arooxa Can we simplify these relationship names? Some general ideas included below:
- "Parent of Repository Collaborator" to "RepositoryCollaborator in the display name
- "RepositoryToOrganization" can be simplified to "OrganizationRepository" 
- "TeamToOrganization" to "OrganizationTeam"
- "UserToOrganizationUser" to "OrganizationUser"


Aim for this is to be consistent with other relationship names, Jira and Salesforce pictured below:
Jira
<img width="894" alt="Screenshot 2024-03-11 at 5 52 03 PM" src="https://github.com/SGNL-ai/catalog-sgnl-templates/assets/126013561/0f71943c-5773-4f5b-aacd-650d9c19b498">
SFDC
<img width="1045" alt="Screenshot 2024-03-11 at 5 52 23 PM" src="https://github.com/SGNL-ai/catalog-sgnl-templates/assets/126013561/404d9e51-2695-4113-8f7d-3614d29cf867">